src/sdl2_compat.c

```diff
@@ -1685,14 +1714,14 @@ static SDL2_Event *Event3to2(const SDL_Event *event3, SDL2_Event *event2)
             motion->state = (Uint8)event3->motion.state;
             motion->x = (Sint32)event3->motion.x;
             motion->y = (Sint32)event3->motion.y;
-            motion->xrel = (Sint32)event3->motion.xrel;
-            motion->yrel = (Sint32)event3->motion.yrel;
+            motion->xrel = (Sint32)AccumulateFloatValueToInteger(&relative_mouse_event_x_frac, event3->motion.xrel);
+            motion->yrel = (Sint32)AccumulateFloatValueToInteger(&relative_mouse_event_y_frac, event3->motion.yrel);
         } else {
             SDL2_MouseMotionEvent *motion = &event2->motion;
             motion->x = (Sint32)event3->motion.x;
             motion->y = (Sint32)event3->motion.y;
-            motion->xrel = (Sint32)event3->motion.xrel;
-            motion->yrel = (Sint32)event3->motion.yrel;
+            motion->xrel = (Sint32)AccumulateFloatValueToInteger(&relative_mouse_event_x_frac, event3->motion.xrel);
```

Comment:
Cast may not be needed here, definitely not needed below casting to int.

Reply:
I'll ditch the casts and see if CI stays happy on all platforms.